docs/expand-pvc.md

```diff
@@ -115,7 +115,7 @@ items:
     conditions:
     - lastProbeTime: null
       lastTransitionTime: "2019-12-19T05:49:39Z"
-      message: Waiting for user to (re-)start a pod to finish file system resize of
+      message: Waiting for user to (re-)start a pod to finish file system expand of
```

Comment:
it should not be changed here see https://github.com/kubernetes-csi/external-resizer/blob/9805ed39a9fe7c533034619b79faef26d1b3afb0/pkg/controller/controller.go#L475